funds-manager/funds-manager-api/src/types/quoters.rs

```diff
@@ -94,13 +94,16 @@ pub struct ExecutionQuote {
     /// The estimated gas for the swap
     #[serde(with = "u256_string_serialization")]
     pub estimated_gas: U256,
```

Comment:
Decided to keep this in case we need it, but we currently do not use it anywhere. Let me know if you think we should remove it for simplicity.